src/Components/Settings/Settings.js

```diff
@@ -2,6 +2,14 @@ import React, { useContext, useEffect } from 'react'
 import style from './Settings.module.scss'
 import { SettingsContext } from '../../Context/SettingsContext'
 import { updateSettings } from '../../apiCalls'
+// import gong from '../../assets/audio/gong.mp3'
+// import goodOldSynths from '../../assets/audio/goodOldSynths.mp3'
+// import pianoDreams from '../../assets/audio/pianoDreams.mp3'
+// import chordCliff from '../../assets/audio/chordCliff.mp3'
+// import levelUp from '../../assets/audio/levelUp.mp3'
+// import birdChord from '../../assets/audio/birdChord.mp3'
+// These ↑ are here in case we want the sounds to play onSelect from the settings menu
```

Comment:
I think this would be such a cool addition!!

I wonder if in the onChange event, we can query for `event.target.value`- and then run a version of the `playAlertSound` function you built in `CountdownTimer` with this value...

Reply:
That was what I was thinking! I tried exporting that `playAlertSound` method to be extra dry, but I was getting a weird react error when I did that. Maybe we can peek at it together!